doc/html/fontstyles.html

```diff
@@ -4,10 +4,14 @@
   <!-- AP: Last modified: 5-Oct-2006 -->
   <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=UTF-8">
   <TITLE>Font Styles in various languages</TITLE>
-  <LINK REL="icon" href="fftype16.png">
-  <LINK REL="stylesheet" TYPE="text/css" HREF="FontForge.css">
+  <LINK REL="icon" href="/assets/img/old/fftype16.png">
+  <LINK REL="stylesheet" TYPE="text/css" HREF="/assets/css/old/FontForge.css">
 </HEAD>
 <BODY STYLE="background: #fff">
+<div style="margin:0; height: 4	em; padding: 0.5em; background: red; color:yellow; text-align:center; font-size:1em; font-family: sans-serif;">
```

Comment:
Same for this

Reply:
Sorry, that was taken from the fontforge.github.io repository … (I didn't check if they differ).